bin/boilerplate.rs

```diff
@@ -241,13 +264,28 @@ impl Harness {
                 },
                 event::Event::MainEventsCleared => {
                     let spawner = task_pool.spawner();
-                    let duration = time::Instant::now() - last_time;
-                    last_time += duration;
-                    let delta = duration.as_secs() as f32 + duration.subsec_nanos() as f32 * 1.0e-9;
 
-                    let update_command_buffers = app.update(&device, delta, &spawner);
-                    if !update_command_buffers.is_empty() {
-                        queue.submit(update_command_buffers);
+                    let delta: f32;
+
+                    #[cfg(target_arch = "wasm32")]
+                    {
+                        let duration = web::now() - last_time;
+                        last_time += duration;
+                        delta = (duration / 1000.0) as f32;
+                    }
+
+                    #[cfg(not(target_arch = "wasm32"))]
+                    {
+                        let duration = time::Instant::now() - last_time;
+                        last_time += duration;
+                        delta = duration.as_secs() as f32 + duration.subsec_nanos() as f32 * 1.0e-9;
+                    }
```

Comment:
I believe it would be better to use [instant](https://crates.io/crates/instant) crate instead of #cfg.

Reply:
web-sys also have implementation for getting time, I just did my own, and then found web-sys one)